src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversed = {};
+
+  for (const key in robot) {
+    if (inversed.hasOwnProperty(robot[key])) {
+      return null;
+    }
+    inversed[robot[key]] = key;
+  }
+
+  const rightRobot = {};
+
+  for (const detail in robot) {
+    if (rightRobot.hasOwnProperty(robot[detail])) {
+      return null;
+    }
+    rightRobot[robot[detail]] = detail;
+  }
+
+  return rightRobot;
```

Comment:
this loop is the same as the previous one, you can remove it and return `inversed` from the function